test/simulator/timestamp/src/include/connection_simulator.h

```diff
@@ -30,8 +30,12 @@
 
 #include <memory>
 #include <vector>
+#include <map>
 
 #include "session_simulator.h"
+#include "timestamp_manager.h"
+
+class timestamp_manager;
```

Comment:
Why do we have this here?

Reply:
Without it I get a `error: 'timestamp_manager' does not name a type` message. Maybe it has something to do with cmake and included things?

Reply:
I still get that error without it so I'm not sure what I'm doing wrong there.

Reply:
[This](https://github.com/wiredtiger/wiredtiger/pull/8116#discussion_r920714793) should fix it